components/tls/pkg/tls/util.go

```diff
@@ -63,3 +67,13 @@ func GetSASLMechanismFromEnv(prefix string) string {
 	}
 	return val
 }
+
+func GetEndpointIdentificationMechanismFromEnv(prefix string) string {
+	val, ok := os.LookupEnv(fmt.Sprintf("%s%s", prefix, EnvEndpointIdentificationMechanismSuffix))
+	if !ok {
+		val = EndpointIdentificationMechanismHTTPS
```

Comment:
Should we default to `https` if the env var is not present? Or would it be better to keep it this way for backwards compatibility?

Reply:
That's what is happening - if not present it will be `https`. 
If it is empty string it will be `None` (which as I understand for Golang is equivalent of empty string). 

In any case I wanted to ensure backwards compatibility